src/Core/BaseObject.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @file src/Core/BaseObject.php
+ */
+namespace Friendica\Core;
+
+if (class_exists('BaseObject')) {
```

Comment:
@annando 
do we still need this this condition?

Reply:
Nope!

Reply:
done

Reply:
ok then in `src /Core/Item.php` is another that can be removed

Reply:
and Core/Conversation.php too?